src/pages/studyView/charts/pieChart/PieChart.tsx

```diff
@@ -1,152 +1,87 @@
-  '#298632', '#dc3922', '#f88588', '#109658',
-  '#990010', '#009916', '#dd4447', '#66aa60',
-  '#b82e9e', '#316365', '#994489', '#22aa69',
-  '#aaaa51', '#66332c', '#e67390', '#8b0777',
-  '#651037', '#329232', '#557486', '#3b3e4c',
-  '#b77372', '#16d690', '#b91310', '#f4358e',
-  '#9c5910', '#a9c493', '#2a773d', '#668d5c',
-  '#bea463', '#0c5952', '#743471', '#743450',
-  '#2986e3', '#dc3914', '#f88503', '#109614',
-  '#990092', '#0099c8', '#dd4476', '#66aa04',
-  '#b82e27', '#316397', '#994495', '#22aa93',
-  '#aaaa14', '#6633c1', '#e67303', '#8b0705',
-  '#651062', '#329267', '#5574a1', '#3b3ea5'
-];
 @observer
 export class PieChart extends React.Component<IPieChartProps, {}> {
 
+    private colorSet:{[id:string]:string} = {};
 
-  constructor(props: IPieChartProps) {
-    super(props);
-  }
+    constructor(props: IPieChartProps) {
+        super(props);
+        this.colorSet = getPieSliceColors(props.data.counts);
+    }
 
-  private events = [{
-    target: "data",
-    eventHandlers: {
-      onClick: () => {
-        return [
-          {
-            target: "data",
-            mutation: (props:any) => {
-              this.props.onUserSelection(props.datum.x);
+    private events = [{
+        target: "data",
+        eventHandlers: {
+        onClick: () => {
+            return [
+            {
+                target: "data",
+                mutation: (props:any) => {
+                    this.props.onUserSelection(props.datum.x);
+                }
             }
-          }
-        ];
-      }
-    }
-  }];
+            ];
+        }
+        }
+    }];
 
-  @computed get totalCount(){
-    return this.props.data.counts.reduce((acc:number,next)=>{
-      return acc+next.count
-    },0)
-  }
+    @computed get totalCount(){
+        return this.props.data.counts.reduce((acc:number,next)=>{
+        return acc+next.count
+        },0)
+    }
 
-  public render() {
-    let data:ClinicalDataCount[] = this.props.data.counts;
-    let chartFilters = this.props.filters;
-    let result =data.map(prop => {
-      let toReturn = {x:prop.value,y:prop.count};
-      if(_.includes(chartFilters,prop.value)){
-        toReturn = Object.assign({}, toReturn, { stroke: "#cccccc", strokeWidth: 3 });
-      }
-      return toReturn;
-    })
-    return (
-        <VictoryPie
-          colorScale={COLORS}
-          containerComponent={<VictoryContainer responsive={false}/>}
-          width={200}
-          height={200}
-          labelRadius={30}
-          padding={35}
-          //to hide label if the angle is too small(currently set to 20 degrees)
-          labels={(d:any) => ((d.y*360)/this.totalCount)<20?'':d.y}
-          data={result}
-          events={this.events}
-          labelComponent={<CustomTooltip />}
-          style={{
-            data: { fillOpacity: 0.9 },
-            labels: { fill: "white" }
-          }}
-        />
-    );
-  }
+    public render() {
+        return (
+        <div className="study-view-pie">
```

Comment:
i think you could put the class on the VictoryContainer itself, which has a property className.